src/backend/login/usingPassport.js

```diff
@@ -28,6 +31,11 @@ passport.use(
     },
     // This function returns 'issuer:' issuer, 'userId:' userId, 'accessToken:' accessToken, 'refreshToken:' refreshToken, 'params:' params
     (req, issuer, userId, profile, accessToken, refreshToken, params, callback) => {
+      log.info('issuer:', issuer);
+      log.info('userId:', userId);
+      log.info('accessToken:', accessToken);
```

Comment:
This is re-adding something we removed in a recent commit.  We don't want to print the accessToken or refreshToken I don't think, cc @Grommers00 to confirm.

Reply:
Oh okay, I added it back during rebase, did not know it was removed on purpose, will remove it.